Automattic/newspack-theme
# Fix: Run screenshots through image optimizer, to reduce size.

### All Submissions:






### Changes proposed in this Pull Request:

Similar to a step used in #718, this PR includes screenshots for the remaining themes that have been run through TinyPNG to reduce their file size. 

Updated screenshots should include the one for the Newspack Theme, Newspack Sacha, Newspack Scott, and Newspack Katharine.

### How to test the changes in this Pull Request:

1. Apply the PR.
2. View each screenshot; confirm they are correct and still look good.

### Other information:
